sys-apps/smartmontools/smartmontools-9999.ebuild

```diff
@@ -84,16 +86,21 @@ pkg_postinst() {
 		local db_path="/var/db/${PN}"
 
 		if [[ -f "${db_path}/drivedb.h" ]] ; then
-			ewarn "WARNING! The drive database file has been replaced with the version that"
-			ewarn "got shipped with this release of ${PN}. You may want to update the"
-			ewarn "database by running the following command as root:"
+			ewarn "WARNING! The existing copy of the drive database has been replaced with the version that"
+			ewarn "was shipped with this release of ${PN}. You may want to update the"
+			ewarn "database by: "
+
+			if ! use update_drivedb ; then
+				ewarn "re-merging ${PN} with USE='update_drivedb', then"
+			fi
+			ewarn "running the following command as root:"
 			ewarn ""
-			ewarn "/usr/sbin/update-smart-drivedb"
+			ewarn "/usr/sbin/${UPDATE_DRIVEDB}"
```

Comment:
This is wrong. The tool is called update-smart-drivedb and not smartmontools-update-drivedb. Please remove this change!